### DISCUSSION TITLE
Restrict lava dumping to claimed land only

### DISCUSSION THREAD

**Discussion by x1p** [GriefPrevention/GriefPrevention#772] [Ideas]
**As** a player you can dump lava anywhere on the world
**I want** to restrict the use of lava only within the claim, something similar to pistons not working outside land claims, and not be able to flow out of the claim.
**Because** Players can ruin the world if dumping lava anywhere random in the world

Optional questions:

**Is your feature request related to a problem? Please describe.**
Especially new players, nothing to lose, dump lava just for fun.

**Describe alternatives you've considered**
Blacklisting lava completely is not an option, it's a part of survival.

**Comment by RoboMWM** [comment]
I think the creative claims mode does this, at least I've seen it for water. I think Big_Scary disabled this for other claim types because it's expensive. But either way should be an addon nonetheless.

**Comment by x1p** [comment]
Yes, but iirc, creative mode restricts everything outside claims.
